examples/hey_fire_fox.py

```diff
@@ -0,0 +1,9 @@
+from howl.client import HowlClient
+
```

Comment:
Can you write some comments about what this file is supposed to do.?
just like how I had it for last scripts that got merged?